go.mod

```diff
@@ -4,110 +4,44 @@ go 1.14
 
 require (
 	cloud.google.com/go/storage v1.12.0
-	dmitri.shuralyov.com/gpu/mtl v0.0.0-20191203043605-d42048ed14fd // indirect
-	github.com/Azure/go-autorest/autorest v0.9.6 // indirect
-	github.com/BurntSushi/xgb v0.0.0-20200324125942-20f126ea2843 // indirect
 	github.com/GoogleCloudPlatform/cloudsql-proxy v1.17.0
-	github.com/Microsoft/go-winio v0.4.16 // indirect
-	github.com/Microsoft/hcsshim v0.8.14 // indirect
-	github.com/PuerkitoBio/goquery v1.5.1 // indirect
-	github.com/antihax/optional v1.0.0 // indirect
 	github.com/aws/aws-sdk-go v1.35.37
-	github.com/bazelbuild/bazelisk v1.7.4 // indirect
 	github.com/bazelbuild/rules_go v0.24.3
-	github.com/bojand/ghz v0.93.0 // indirect
```

Comment:
There are certain packages that are only referenced by the buildbuddy-internal repo and running `go mod tidy` on the public repo will remove them since it does not look at the go files in the internal repo. I think such packages were added to this go.mod file by hand. I'm not sure if we have a way to fix this until we consolidate the two repos.

Reply:
Perhaps one workaround is to have a dummy Go file in the public repo solely to import packages that are used in the internal repo. That should be enough to trick `go mod tidy` into doing the right thing.

Reply:
I talked with Tyler, and he doesn't seem excited about dummy files at all, so we may wait for the unification and then start running `go mod tidy`, possibly with a script to set up temporary dummy files if they end up being necessary. I found a bunch of protos that weren't declaring the correct package, so it's possible that that fix will handle the proto side of things? maybe. still would need a dummy file for bundle, though.

Reply:
For bundle, I think we could move the go file to bundle/bundle.go but keep the go_library rule in the top level BUILD file.

Reply:
so all the protos were fixed by the package corrections after all (after re-running your symlink creation script). unfortunately, because of how embed works, bundle absolutely has to be in the root dir to do what we need it to do. Still, that's just one dummy file instead of like 30, which might be more tolerable.

Reply:
maybe the imports file you suggested isn't so bad, also? as it actually documents which deps are needed by the external module instead of throwing it in with everything else from anyone who ever ran `go get` in the repo.